examples/multisig/scripts/common/common.js

```diff
@@ -4,6 +4,6 @@ exports.executeTransaction = async function (deployer, txnParams){
     try {
        await executeTransaction(deployer, txnParams);
     } catch (e) {
-        console.error('Transaction Failed', e.response.error);
+        console.error('Transaction Failed', e.response ? e.response.error : e);
```

Comment:
I think you should be able to write:
```
        console.error('Transaction Failed', e.response.?error : e);
```

Reply:
It's showing an error: `Identifier expected`
![image](https://user-images.githubusercontent.com/33264364/100613184-1f3ad080-333a-11eb-90e6-da7cd47bdbf3.png)